source/pages/pki/pkiRotate.rst

```diff
@@ -4,128 +4,144 @@
-     :emphasize-lines: 4-5, 11-13
 
-     privateEncryptionKeyConfiguration:
-      keyUri: deploy/keys/test_rp_msa_encryption_old.pk8
+    signingKeys:
+      primary:
+        publicKey:
+          certFile: msa_signing_2016.crt
+          name: 2016 MSA Signing Key
+        privateKey:
+          keyFile: msa_signing_2016.pk8
+      secondary:
+        publicKey:
+          certFile: msa_signing_2017.crt
+          name: 2017 MSA Signing Key
+        privateKey:
+          keyFile: msa_signing_2017.pk8
 
-     privateSecondaryEncryptionKeyConfiguration:
-      keyUri: deploy/keys/test_rp_msa_encryption_new.pk8
+2. Run the MSA to publish the new signing certificate (2017) to its metadata.
 
-     publicEncryptionKeyConfiguration:
-      keyUri: deploy/keys/test_rp_msa_encryption_old.crt
-      keyName: http://www.test-rp-ms.gov.uk/SAML2/MD
+3. Wait for the VSP to load the MSA metadata. The VSP periodically refreshes its metadata and will log when it has done it. Once it loads the new metadata, your VSP trusts assertions signed with the new (2017) MSA signing key.
 
-     publicSecondaryEncryptionKeyConfiguration:
-      keyUri: deploy/keys/test_rp_msa_encryption_new.crt
-      keyName: http://www.test-rp-ms.gov.uk/SAML2/MD
- 
-  The MSA can now use both the new and old keys to decrypt SAML messages.
+4. Delete the ``signingKeys.primary`` section and rename ``signingKeys.secondary`` to ``signingKeys.primary``. The MSA now signs the assertions with the new (2017) key.
 
-  .. note:: While both keys are in use, you may see error messages in the logs with the description 'Unwrapping failed'. These messages appear because the MSA attempts to decrypt the SAML message using each key in turn. You can safely ignore these messages. However, do not ignore any other error messages related to SAML decryption.
+5. Inform the GOV.UK Verify Team that the new key is live.
 
-2. Send the new certificate to the GOV.UK Verify operations team.
-3. The GOV.UK Verify operations team replaces the old certificate with the new certificate on the GOV.UK Verify hub. The GOV.UK Verify hub now uses the new key to encrypt SAML messages for your service.
-4. Promote the secondary private encryption key and encryption certificate to the primary position. 
-5. Remove the old private encryption key and encryption certificate:
 
+.. _rotateVSP_encKey:
 
-  .. code-block:: yaml
+Rotate your VSP encryption key
+-----------------------------------
 
-     privateEncryptionKeyConfiguration:
-      keyUri: deploy/keys/test_rp_msa_encryption_new.pk8
+These instructions apply to you if you're using the Verify Service Provider.
 
-     publicEncryptionKeyConfiguration:
-      keyUri: deploy/keys/test_rp_msa_encryption_new.crt
-      keyName: http://www.test-rp-ms.gov.uk/SAML2/MD
+#. Add the new VSP private encryption key you've generated to the ``samlSecondaryEncryptionKey`` field in your VSP configuration. Your VSP can now use both the new and old keys to decrypt SAML mesasges.
```

Comment:
I know this is valid RST but I feel like we should be consistent about using numbers (1., 2., 3.), a repeated number (1., 1., 1.) and hashes (#., #., #.)